grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -735,3 +686,88 @@ def _GetTypeName(
     return cast(str, primitive_types[cls]["name"])
 
   return str(cls)  # Cover `BinaryStream` and `None`.
+
+
+def _GetReferenceObject(type_name: str) -> SchemaReference:
+  """Get a `Reference Object` that points to a schema definition.
+
+  All types (including protobuf primitives) are expected to have been
+  previously defined in the `components` field of the root `OpenAPI Object`
+  and are used via OpenAPI references.
+
+  Args:
+    type_name: The name of the type for which an OpenAPI `Reference Object`
+      will be created and returned.
+
+  Returns:
+    An OpenAPI `Reference Object` representing the path to the actual OpenAPI
+    schema definition of the selected type.
+  """
+  return {
+    "$ref": f"#/components/schemas/{type_name}"
```

Comment:
Missing trailing comma (autoformatter would re-format it to ` return {"$ref": f"..."}`, trailing comma ensure that this does not happen and make diffs nicer).

Reply:
Added them. Thank you!